api/app/signals/apps/feedback/models.py

```diff
@@ -0,0 +1,51 @@
+import secrets
+import uuid
+from datetime import timedelta
+
+from django.contrib.gis.db import models
+from django.utils import timezone
+
+from signals.apps.feedback.managers import FeedbackManager
+from signals.apps.signals.models import Signal
+
+FEEDBACK_EXPECTED_WITHIN_N_DAYS = 14  # move to general settings file
+
+
+def generate_token():
+    """Use secrets module of Python to generate a UUID object."""
+    return uuid.UUID(hex=secrets.token_hex(16))
+
+
+class StandardAnswer(models.Model):
+    is_visible = models.BooleanField(default=True)
+    is_satisfied = models.BooleanField(default=True)
+    text = models.TextField(max_length=1000)
+
+
+class Feedback(models.Model):
+    # Bookkeeping
+    token = models.UUIDField(db_index=True, primary_key=True, default=generate_token)
+    _signal = models.ForeignKey(Signal, on_delete=models.CASCADE, related_name='feedback')
+    created_at = models.DateTimeField(auto_now_add=True)
+    submitted_at = models.DateTimeField(editable=False, null=True)
+
+    # Fields that we expect Reporter to fill out
+    is_satisfied = models.BooleanField(null=True)
+    allows_contact = models.BooleanField(default=False)
+    text = models.TextField(max_length=1000, null=True, blank=True)
+    text_extra = models.TextField(max_length=1000, null=True, blank=True)
+
+    objects = models.Manager()
```

Comment:
No need to set this explicit because we do not override the default objects behaviour

Reply:
Same as the `signals.apps.signals.models.Signal` model.